CONTRIBUTING.md

```diff
@@ -1,9 +1,9 @@
 # Contributing to Kubernetes Documentation
 
-**First off, thanks for taking the time to contribute!**
+**First of all, thanks for taking the time to contribute!**
 
 The following is a set of guidelines for contributing to Kubernetes documentation, hosted at [Kubernetes.io](http://kubernetes.io/).
-These are just guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+These are just guidelines, not rules. Use your best judgement, and feel free to propose changes to this document in a pull request.
```

Comment:
Judgment is also valid spelling form, there’s two possibilities of this word whether it’s judgment or judgement.